scripts/output_env.sh

```diff
@@ -41,81 +45,100 @@ print_version()
         VERSION_STR=`echo "$VERSION_STR" | $FILTER`
     done
 
-    echo "* ${BIN##*/}: $BIN: $VERSION_STR"
+    echo " * ${BIN##*/}$VARIANT: ${BIN} : ${VERSION_STR} "
 }
 
-print_version "uname" "-a" ""
+echo "Platform:\n"
+if `type "lsb_release" > /dev/null 2>&1`; then
+    echo "Linux variant"
+    lsb_release -d -c
+else
+    echo "Unknown Unix variant"
+fi
+
 echo
 
+print_version "uname" "-a" ""
```

Comment:
It would make more sense to put the `uname` output at the beginning of the “Platform” section, before adding more precise information about the Linux distribution.

Reply:
I'm not sure about that. For me Distro version comes before kernel version, which is what `uname` gives you - although it does also give architecture which arguably belongs first.

I can change it if you feel strongly, but I think I prefer it this way around.

Reply:
The output of `uname -a` contains information that's more generic than the distro (`uname -s`), information that's more specific (`uname -rv` — which often contains distribution information, but might be misleading if the distro is running under some userland virtualization mechanism with a kernel from a different distribution), and information that's orthogonal (`uname -mn`). I was thinking of `uname -s` first, then the distro. But it doesn't really matter. I'm happy with the code as it is.